packages/design-system/src/components/Badge/Badge.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import cx from 'classnames';
+import styles from './style.scss';
+
+function Badge({ children, className: extraClassName, secondary, ...props }) {
+  const className = cx(styles.badge, {
+    [styles['badge--secondary']]: secondary
+  });
+
+  return (
+    <span className={className} {...props}>
```

Comment:
If you don't want to include className in the component api you need to change the order:
```suggestion
    <span {...props} className={className}>
```

Reply:
It's not needed with the way it's currently done.

Reply:
ok